ci/update-deps.js

```diff
@@ -124,6 +135,191 @@ async function maybeUpdateVersion(plugin, minorVersion, version) {
     }
 }
 
+/**
+ * Drafts a post on Lobby VIP for Jetpack releases.
+ *
+ * @param {string} version - The version of Jetpack being released
+ * @param {string} type - Type of post being drafted. Accepted values: beta, release
+ * @returns {boolean} Whether the post was successfully drafted or not
+ */
+async function draftJPPost( version, type ) {
+    const allowedTypes = [ 'beta', 'release' ];
+    if ( ! allowedTypes.includes( type ) ) {
+        return false;
+    }
+
+    if ( type === 'beta' && ! version.includes( 'beta' ) ) {
+        return false;
+    }
+
+    const changelog = await fetchChangelog( version );
+    const section = extractChangelogSection( changelog, version, type );
+
+    if ( section ) {
+        let title = '';
+        let content = '';
+        if ( type === 'beta' ) {
+            title = `Call for Testing: Jetpack ${version}`;
+            content = createJPBetaPostContent(version, section);
+        } else {
+            title = `New Release: Jetpack ${version}`;
+            content = createJPReleasePostContent(version, section);
+        }
+        const post = createJPPost( title, content );
+        if ( post ) {
+            const postUrl = `https://lobby.vip.wordpress.com/wp-admin/post.php?post=${post.id}&action=edit`
+            pingSlack(`@vip-cantina-team Jetpack ${version} draft created for review: ${postUrl}. Don't forget to deploy first before publishing!`);
+            return true;
+        } else {
+            pingSlack(`@vip-cantina-team Error creating Jetpack ${version} draft.`);
+            return false;
+        }
+    }
+}
+
+/**
+ * Gets the entire changelog file contents of Jetpack from GitHub.
+ *
+ * @async
+ * @param {string} version - The version of Jetpack being released
+ * @returns {Promise<Object>} - The response data from the API
+ */
+async function fetchChangelog(version) {
+    const url = `https://raw.githubusercontent.com/Automattic/jetpack-production/${version}/CHANGELOG.md`;
+
+    try {
+      const response = await axios.get(url);
+      return response.data;
+    } catch (error) {
+      throw error;
+    }
```

Comment:
```suggestion
    const response = await axios.get(url);
    return response.data;
```


No need to catch an error if you are going to rethrow it without any handling.